### ISSUE TITLE
CTD should not have "Treats"

### ISSUE THREAD

**Issue by cbizon** [NCATSTranslator/Feedback#978]
The question is "What treats polymyositis?" in PROD
PK = 7aa99faa-37e6-4a63-a604-7d3e73b908d3

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/14f819ac-69c7-4d46-9a33-9a8d70b773a4">

That treats edge seems to be coming from CTD via BTE:

<img width="628" alt="image" src="https://github.com/user-attachments/assets/8e9579ab-621b-4f0e-9d34-8e5ca3294e0c">

CTD doesn't really report "Treats" as we understand it.  It looks like it's coming via molepro.  And I'm seeing the same edge in CI.

**Comment by cbizon** [comment]
@mbrush

**Comment by vdancik** [comment]
@cbizon CTD provides "therapeutic" flag for chemicals:

![image](https://github.com/user-attachments/assets/6ac96f34-3724-4943-96b2-cb3654eeb6c8)

**Comment by cbizon** [comment]
That's true, but I don't think that CTD's "Therapeutic" rises to the level of `treats` in biolink:

>‘ Treats’ edges should be asserted (knowledge_level: assertion) only in cases where there
      is strong supporting evidence - i.e. in some population(s) the intervention is approved for
      the condition, passed phase 3 or in phase 4 trials for the condition, or is an otherwise 
      established treatment in the medical community (e.g. a widely-accepted or formally recommended 
      off-label use). In the absence of such evidence, weaker predicates should be used in 
      asserted edges (e.g. ‘in clinical trials for’ or ‘beneficial in models of’).

I think a mapping of `treats_or_studied_or_applied_to_treat` would be more appropriate (and is how we ingest CTD in robokop).

**Comment by mbrush** [comment]
That was also my recommendation based on what you told me about the provenance of CTD 'T' annotations, and the level of confidence we have that they represent true 'treatments' that fit our definition and implementation guidance around the 'treats' predicate.   @vdancik if you think differently (i.e. that CTD 'T' annotations are strong enough to warrant a 'treats' assertion), happy to have a conversation about this.
